modules/aks/starterkit/buildingblock/main.tf

```diff
@@ -140,9 +143,14 @@ data "meshstack_tenant_v4" "aks-prod" {
   }
 }
 
-resource "meshstack_building_block_v2" "github_actions_dev" {
-  depends_on = [meshstack_building_block_v2.repo, meshstack_tenant_v4.dev]
+# only to fetch data from GitHub building block after creation
```

Comment:
d: why would i need a data object to retrieve data if i already have the resource? shouldn't the resource contain all the data already?